root/user/UserProfile.js

```diff
@@ -709,6 +722,58 @@ const UserProfileStatistics = ({
           )}
         </tbody>
       </table>
+
+      {hasPublicTags || hasPublicRatings ? (
+        <table
+          className="statistics"
+          title={l('This table shows a summary ' +
+                  'of secondary data added by this editor.')}
+        >
+          <thead>
+            <tr>
+              <th colSpan="2">
+                {l('Tags and ratings')}
+              </th>
+            </tr>
+          </thead>
+          <tbody>
+            {hasPublicTags ? (
+              <>
+                <UserProfileProperty name={l('Tags upvoted')}>
+                  {$c.user && upvotedTagCount > 0 ? exp.l(
```

Comment:
If I have no upvoted tags and no ratings, it shows a "Ratings: 0" row but nothing for tags (which I guess only seems odd since the heading is "Tags and ratings.") Maybe we should always show the "Tags upvoted" row even if it's 0?

Reply:
Actually, this is probably because the materialized columns in my sample data say I have ratings. I see it's not supposed to show anything at all!

Reply:
Are your tags public? My idea was that it would show 0 if they are public but you have none, and not show at all if they are set to private. But maybe my check doesn't work as well as I thought (I only tested it with ratings). Will look.

Reply:
Yeah, on review, we do need to set a default of 0 for the tags if tags are public (or you're viewing your profile) in Perl, they otherwise remain undef if there's 0.

Reply:
Did you push the change? I don't see anything in the PR timeline.

Reply:
I did now, sorry :)